container/container.bzl

```diff
@@ -145,3 +148,10 @@ py_library(
       remote = "https://github.com/google/subpar",
       commit = "7e12cc130eb8f09c8cb02c3585a91a4043753c56",
     )
+
+  if "structure_test" not in excludes:
```

Comment:
Generally we don't define `contrib` deps here.  Can you create a `repositories()` function in `test.bzl` for this?

What transitive deps does this have?  I also worry that we're introducing an implicit dependency on `rules_go`, which cannot easily be encapsulated into one of these function (further motivation to separate it's setup).

Reply:
Ah yep, I'll pull it out. All of the deps in the container-structure-test are vendored in, and `rules_do` isn't one of them, so I think we should be good here.